modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java

```diff
@@ -37,6 +38,14 @@ public boolean isDisabled(Object object) {
 			return true;
 		}
 
+		UserNotificationEvent storedUserNotificationEvent =
+			UserNotificationEventLocalServiceUtil.fetchUserNotificationEvent(
+				userNotificationEvent.getUserNotificationEventId());
+
+		if (storedUserNotificationEvent == null) {
+			return true;
+		}
+
```

Comment:
Can you rename this to `userNotificationEvent`? Unless there's a reason to disambiguate, we usually use the raw type name as variable name.

Reply:
We have another `userNotificationEvent` in some lines above (line 33-34 the link isn't working well). 
Do you have a better name in mind?

Reply:
I didn't see that :facepalm: